node/node.go

```diff
@@ -1559,7 +1559,8 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		service := v2alpha1.NewNetworkService(
 			app.clock.GenesisTime(),
 			app.Config.Genesis.GenesisID(),
-			app.Config.LayerDuration)
+			app.Config.LayerDuration,
+			app.Config.POST.LabelsPerUnit)
```

Comment:
Would it make sense to just pass the whole `app.Config` object instead of just 3 fields from it?